docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -21,6 +21,21 @@ excerpt: ""
 description: "Troubleshooting Session Replay-specific Issues"
 ---
 
+<Expandable title="Replay does not look correct" permalink>
```

Comment:
Maybe this could be more specific? Like "Replay does not match my site" (or application)

Reply:
```suggestion
<Expandable title="Replay does not match my application" permalink>
```